### ISSUE TITLE
How did you make the iOS shortcut open up in a new app?

### ISSUE THREAD

**Issue by IoIxDButStronger** [gameplaycolor/gameplaycolor#96]
Something I've noticed with this app is that unlike traditional ios emulators, which run via some kind of jailbreak app, this app is installed via Safari's "home button" feature. This is a pretty common strategy, I know, but what separated this app from other apps like this was that

* It opened as its own app, completely separate from Safari.

* It's clearly running on Safari, but it has the address bar minimized/hidden, something that I can't seem to accomplish on my own website.

How do you do this? It's almost like it's a separate app, but it's clearly running in Safari. How can I do this myself?

**Comment by bessx** [comment]
It's called a Progressive Web App (PWA).

More info here: https://developers.google.com/web/progressive-web-apps/

**Comment by jbmorley** [comment]
@IoIxDButStronger If I remember correctly, the key is to add something similar to the following to the HTML head of your page:

```
    <meta name="viewport" content="user-scalable=no, initial-scale=1.0, maximum-scale=1.0">
    <meta name="apple-mobile-web-app-capable" content="yes">
    <meta name="apple-mobile-web-app-status-bar-style" content="black">
```

This will prevent scaling, and allow it to open as a separate app (rather than a link in Safari).

Closing this issue as 'invalid' as it's not actually a problem with Game Play. 🙃

**Comment by IoIxDButStronger** [comment]
I vaguely remember being told this back then by somebody else as well, and for whatever reason I could never get it to work. It's been so long that I don't remember why it didn't work, it just...didn't work. A month later, I got fed up with the ability to do anything cool with my iPhone without jailbreaking and I switched to Android.

Ironically, last month my Android broke and I was forced to go back to an iPhone until I can fix my other phone, and I ended up just jailbreaking my phone. It's not really as damaging or hard as I once thought it was, and/or as it probably once was.
